conf/common/test/slurm_ray_container_vllm.toml

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: NVIDIA CORPORATION & AFFILIATES
+# Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
For new files please set a single year value (diagnostic we have today is misleading)